runtime/near-vm-logic/tests/test_misc.rs

```diff
@@ -0,0 +1,51 @@
+mod fixtures;
```

Comment:
We need to test:
* `storage_iter_range`;
* Using multiple iterators at the same time;
* Making iterators invalid and checking that they return error on access;
* Corner cases:
   * Attempt to create an empty iterator (when we create iterator from prefix and there are no elements with this prefix);
   * Attempt to create iterator from reversed range (where start > end);
   * Situation when we use `storage_iter_range(start, end)` but there is not element that matches `start` exactly and so the first element is the one right after `start`;
* General error cases described here: https://nearprotocol.github.io/nomicon/Runtime/BindingsSpec/BindingsSpec.html
E.g. try causing `GasExceeded` or `GasLimitExceeded` errors.
* Specific error cases described in each API: https://nearprotocol.github.io/nomicon/Runtime/BindingsSpec/TrieAPI.html
E.g. try causing `MemoryAccessViolation` by exceeding memory allocated to registers.


If possible we need to run each test with `MockedExternal` and `RuntimeExt`. This might require linking to `runtime/runtime` from tests. We need the former one because that's what Rust contract unit tests are using and we need the latter one because that's what we use in real network.